force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +390,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
+        Map<String, Integer> fruitMap = new Map<String, Integer>();
 
         // Add key-value pairs to the Map
         // add Apples => 5
         // add Oranges => 10
         // add Bananas => 15
         // add Pears => 20
         // add Grapes => 25
-
-        return null; // Replace null with the variable you used to store the result
+        fruitMap.put('Apples', 5);
+        fruitMap.put('Oranges', 10);
+        fruitMap.put('Bananas', 15);
+        fruitMap.put('Pears', 20);
+        fruitMap.put('Grapes', 25);
+      
+        return fruitMap; 
     }
```

Comment:
A naming convention I like is value by key for a Map. Rather than fruitMap I would call it fruitQuantitiesByName

You could also use the Map constructor  for this if you like. 

'''
        return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };
```